packages/orchestration/src/facade.js

```diff
@@ -182,7 +199,13 @@ export const makeOrchestrationFacade = ({
             return makeLocalChainFacade(localchain);
           }
 
-          return makeRemoteChainFacade(name, {
+          const chainInfo = chainInfos.has(name)
+            ? chainInfos.get(name)
+            : // @ts-expect-error may be undefined
+              wellKnownChainInfo[name];
```

Comment:
well known chain should come from `agoricNames`, yes?

Reply:
eventually, yes. that's https://github.com/Agoric/agoric-sdk/issues/9063. I'll add a TODO